google-cloud-spanner/src/test/java/com/google/cloud/spanner/SpannerGaxRetryTest.java

```diff
@@ -206,6 +212,11 @@ private void warmUpSessionPool(DatabaseClient client) {
 
   @Test
   public void singleUseTimeout() {
+    if (isMultiplexedSessionsEnabled()) {
+      // for multiplexed sessions CreateSessions RPC is already completed during start-up
+      // hence, we are setting a strict delay with the next RPC
+      mockSpanner.setExecuteStreamingSqlExecutionTime(ONE_SECOND);
```

Comment:
In order to replicate the original behavior of this test, you should set `mockSpanner.setCreateSessionExecutionTime(ONE_SECOND)`.

Reply:
The regular sessions path uses `setMinSessions=0` as a workaround to prevent `BatchCreateSessions` RPC during `@Setup`. This ensures that during the test, the `BatchCreateSessions` RPC are invoked and they timeout.

For multiplexed session, the creation only happens during app initialization. Hence during `@Setup` itself the multiplexed session creation call will go. So there is no point of setting `setCreateSessionExecutionTime` within the test as by then mock spanner already would have received the call to create session and returned a response.

So given multiplexed session creation needs to be guaranteed within these tests, we would have to set the delay on whatever the next RPC is and then verify the GAX retry on the same.